tensorrt_llm/_torch/pyexecutor/model_loader.py

```
@@ -1338,22 +1347,29 @@ def abort_update_weights(self) -> None:
+
+        A second call is a no-op because both handles are dropped. Cleanup is
+        best effort so shutdown and destructor paths do not propagate backend
+        failures.
         """
         if self._gms_backend is not None:
             self._gms_backend.cleanup()
             self._gms_backend = None
+        if self._checkpoint_loader is not None:
+            try:
+                self._checkpoint_loader.cleanup()
+            except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)